app/dispatches/players/enlist.rb

```diff
@@ -15,13 +13,13 @@ def initialize(game:, player:)
 
     def assign_game_to_player!
       @player = Player.create!(nickname: nickname, game: game, life: Player::LIFE)
-      @game.update(state: 'running')
+      game.update(state: 'running')
```

Comment:
Tell don't ask: is it too early to call for `Game#start!`? Or, maybe, even `Game#add_player` (which would, in turn, start the game/change its state)?